From bd8ceb4a7b985b99c86bc37a2f9629e04bd7a68a Mon Sep 17 00:00:00 2001
From: Saurabh Sengar <saurabh.singh@xilinx.com>
Date: Thu, 1 Mar 2018 14:32:01 +0530
Subject: [PATCH 0309/1851] dt-bindings: display: xlnx: Modifing mixer
 compatible string

commit 90a2ca542cf533b1b0d0d79e91891b3a46b42523 from
https://github.com/Xilinx/linux-xlnx.git

Adding IP version name in compatible string.

Signed-off-by: Saurabh Sengar <saurabh.singh@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 .../devicetree/bindings/display/xlnx/xlnx,mixer.txt         | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/Documentation/devicetree/bindings/display/xlnx/xlnx,mixer.txt b/Documentation/devicetree/bindings/display/xlnx/xlnx,mixer.txt
index b72c0cf77ade..c3338dd38f23 100644
--- a/Documentation/devicetree/bindings/display/xlnx/xlnx,mixer.txt
+++ b/Documentation/devicetree/bindings/display/xlnx/xlnx,mixer.txt
@@ -1,4 +1,4 @@
-Device-Tree bindings for Xilinx Video Mixer IP core
+Device-Tree bindings for Xilinx Video Mixer IP 3.0 core
 
 The IP core provides a flexible video processing block for alpha blending
 and compositing multiple video and/or graphics layers.
@@ -8,7 +8,7 @@ cores (through AXI4-Stream interfaces) is provided.
 The Video Mixer always has one streaming input layer, known as master layer.
 
 Required properties:
- - compatible: Should be "xlnx,mixer".
+ - compatible: Should be "xlnx,mixer-3.0".
  - reg: Base address and size of the IP core.
  - interrupts: Interrupt number.
  - interrupts-parent: phandle for interrupt controller.
@@ -37,7 +37,7 @@ Optional properties:
  - logo: logo layer
 Example:
 	v_mix_0: v_mix@80100000 {
-		compatible = "xlnx,mixer";
+		compatible = "xlnx,mixer-3.0";
 		interrupt-parent = <&gic>;
 		interrupts = <0 93 4>;
 		reg = <0x0 0x80100000 0x0 0x80000>;
-- 
2.31.1

